### ISSUE TITLE
Service --guid returns full description

### ISSUE THREAD

**Issue by bobbyhubbard** [cloudfoundry/cli#1271]
## Command  
`cf service myservice --guid`

## What occurred
The response now includes a full description of the service in addition to the guid. Scripts expecting the guid are now failing. 

```
hubbardr@ola-hdhd942:~$ cf service secure-credentials --guid
Showing info of service myservice in org myorg / space myspace as r@foo.com...

7cf8e029-2594-4bf7-aeb2-789e22082cee
```
## What you expected to occur
Prior to 6.33, it only returned the guid. `cf app myapp -guid` still returns only the guid. 
## CLI Version  
`cf version 6.33.0+a345ea34d.2017-11-20`

## Platform & Shell Details  
Ubuntu 14.04.3 64-bit with gnome-terminal

**Comment by dkoper** [comment]
Oh no! We'll fix it.

**Comment by dkoper** [comment]
It's fixed in our edge binaries:
https://github.com/cloudfoundry/cli#edge-binaries

Are you OK using a previous cf CLI release or edge binary until our next release (early Dec.)?

**Comment by bobbyhubbard** [comment]
Certainly! Thanks for the quick turnaround.

**Comment by dkoper** [comment]
Fixed in cf CLI 6.33.1. Thanks again for the report.
